movie_REST_API/movie_REST_API/Controllers/InfoMovieViewController.swift

```diff
@@ -233,25 +227,31 @@ final class InfoMovieViewController: UIViewController {
     }
 
     private func loadMoviesData() {
-        actorView.fetchMoviesData(id: idNew) { [weak self] in
+        actorViewModel?.fetchActorsData(id: movie?.id, completion: { [weak self] _ in
             DispatchQueue.main.async {
                 self?.imageCollectionView.reloadData()
             }
-        }
+        })
     }
-    
-    @objc func goToWebButtonAction(sender: UIButton) {
-        let wkWebVC = WKWebViewController()
-        wkWebVC.id = idNew
-        navigationController?.pushViewController(wkWebVC, animated: true)
+
+    @objc private func starAction() {
+        if isPressed == false {
+            showAlert(title: Constant.addFavouriteString, message: Constant.emptyString, handler: nil)
+            navigationItem.rightBarButtonItem?.image = UIImage(systemName: Constant.starFillImageName)
+            isPressed = true
```

Comment:
```suggestion
            isPressed = true
```
isPressed во вью модель вынести